build/app.js

```diff
@@ -58410,13 +58410,21 @@ app.controller('PeersCtrl', ['$scope', function ($scope) {
   $scope.status = "Loading Peers..."
 
   Peers.fetch().then(function(peers) {
```

Comment:
I meant inside fetch() so results are always sorted. Either way is fine with me, and sorting in the controller definitely makes sense too.